pdm/test/action.yml

```diff
@@ -59,3 +59,39 @@ runs:
     - name: Add coverage report to summary
       run: echo -e ${{ steps.coverage.outputs.summaryReport }} >> $GITHUB_STEP_SUMMARY
       shell: bash
+
+    - name: Detect Allure report presence
+      id: check-allure
+      continue-on-error: true
+      run: |
+        [ -d reports/allure ] && REPORT='true' || REPORT='false'
+        echo "has_report=${REPORT}" >> $GITHUB_OUTPUT
+        echo "path_suffix=${{ github.ref_name == 'main' && '' || format('@{0}', github.ref_name) }}" >> $GITHUB_OUTPUT
+      shell: bash
+
+    - name: Publish Allure report
+      id: allure
+      if: steps.check-allure.outputs.has_report
+      continue-on-error: true
+      uses: LedgerHQ/send-to-allure-server-action@2.1.2
+      with:
+        allure-server-url: "https://vault.allure.green.ledgerlabs.net"
+        build-name: ${{ github.workflow }}
+        build-url: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        username: ${{ vars.ALLURE_USERNAME }}
```

Comment:
I don't have access to the variables available on this repo, but I guess this OK

Reply:
Yes, vars & secrets are only available on private repositories by default

Reply:
Added a check for variable presence